### PR TITLE
Dropdown problem Markdown and OLX changes

### PR DESCRIPTION
**Pull request by lamagnifica**
## [DOC-3223](https://openedx.atlassian.net/browse/DOC-3223)

Updates dropdown problem Markdown and OLX to include label and description syntax and OLX reorganization.
### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 
- [ ] Subject matter expert: @muzaffaryousaf (and team as makes sense)
- [ ] Subject matter expert: @cptvitamin 
- [ ] Doc team review (dev edit): @catong 
- [ ] Product review: @sstack22 
- [ ] Partner support: 
- [ ] PM review: 

FYI: @pdesjardins @jaakana @mmacfarlane @cahrens 
### HTML Draft

http://draft-dropdown-problem-component-2016-08.readthedocs.io/en/latest/exercises_tools/dropdown.html
### Testing
- [ ] Ran ./run_tests.sh without warnings or errors
### Sandbox

https://studio-tnl4972.sandbox.edx.org
### Post-review
- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

**Comment by muzaffaryousaf** [comment]
@ehteshamkafeel Can you review this PR since you made changes for drop down.

File: en_us/shared/exercises_tools/dropdown.rst [hunk 1]
```diff
@@ -6,176 +6,207 @@ Dropdown Problem
...
+  What type of data are the following?
+
+  >>Age:<<
+  [[Nominal, Discrete, (Continuous)]]
+  >>Age, rounded to the nearest year:<<
+  [[
+  Nominal
+  (Discrete)
+  Continuous
+  ]]
```
Comment (muzaffaryousaf) [inline en_us/shared/exercises_tools/dropdown.rst:58]: Missing `---` between multiple questions.

File: en_us/shared/exercises_tools/dropdown.rst [hunk 2]
```diff
@@ -6,176 +6,207 @@ Dropdown Problem
...
+  or new lines.
+
+The OLX markup for this example problem follows.
+
+.. code-block:: xml
+
+  <problem>
+    <p>What type of data are the following?</p>
+
+    <label>Age:</label>
```
Comment (muzaffaryousaf) [inline en_us/shared/exercises_tools/dropdown.rst]: This label should come inside respective `<optionresponse>`

File: en_us/shared/exercises_tools/dropdown.rst [hunk 3]
```diff
@@ -6,176 +6,207 @@ Dropdown Problem
...
+    <optionresponse>
+      <optioninput options="('Nominal','Discrete','Continuous')" correct="Continuous"></optioninput>
+      </optionresponse>
+    <label>Age, rounded to the nearest year:</label>
```
Comment (muzaffaryousaf) [inline en_us/shared/exercises_tools/dropdown.rst]: label should come inside respective `<optionresponse>`

File: en_us/shared/exercises_tools/dropdown.rst [hunk 4]
```diff
@@ -6,176 +6,207 @@ Dropdown Problem
...
+    <optionresponse>
+      <optioninput options="('Nominal','Discrete','Continuous')" correct="Discrete"></optioninput>
+      </optionresponse>
+    <label>Life stage - infant, child, and adult:</label>
```
Comment (muzaffaryousaf) [inline en_us/shared/exercises_tools/dropdown.rst]: This label should come inside respective `<optionresponse>`

File: en_us/shared/exercises_tools/dropdown.rst [hunk 5]
```diff
@@ -6,176 +6,207 @@ Dropdown Problem
...
+    <p>What type of data are the following?</p>
+
+    <label>Age:</label>
+    <optionresponse>
+      <optioninput options="('Nominal','Discrete','Continuous')" correct="Continuous"></optioninput>
+      </optionresponse>
+    <label>Age, rounded to the nearest year:</label>
+    <optionresponse>
+      <optioninput options="('Nominal','Discrete','Continuous')" correct="Discrete"></optioninput>
+      </optionresponse>
```
Comment (muzaffaryousaf) [inline en_us/shared/exercises_tools/dropdown.rst:82]: This is off-indent.

File: en_us/shared/exercises_tools/dropdown.rst [hunk 6]
```diff
@@ -346,32 +377,31 @@ Template
...
-    <hint>Hint 1</hint>
-    <hint>Hint 2</hint>
-    <hint>Hint 3</hint>
-  </demandhint>
   <solution>
     <div class="detailed-solution">
       <p>Explanation or Solution Header</p>
       <p>Explanation or solution text</p>
     </div>
   </solution>
```
Comment (muzaffaryousaf) [inline en_us/shared/exercises_tools/dropdown.rst]: `solution` is also a child of `</optionresponse>`. Can you indent `solution` too.

File: en_us/shared/exercises_tools/dropdown.rst [hunk 7]
```diff
@@ -195,84 +226,75 @@ For example, the following problem has feedback for each possible answer.
...
+    <option correct="False">Option Label
+      <optionhint>Feedback for when learner selects this answer.</optionhint>
     </option>
 
 For example, the following problem has feedback for each answer.
 
 .. code-block:: xml
 
+  <problem>
+  <label>A/an ________ is an example of a vegetable.</label>
```
Comment (muzaffaryousaf) [inline en_us/shared/exercises_tools/dropdown.rst]: label should come inside the particular response type.

Comment (muzaffaryousaf) [inline en_us/shared/exercises_tools/dropdown.rst]: Also the children of particular tag should have a tab. Please add tag to children for proper indentation.

File: en_us/shared/exercises_tools/dropdown.rst [hunk 8]
```diff
@@ -6,176 +6,207 @@ Dropdown Problem
...
 ::
 
+  [[
+  Wrong Answer {{Feedback for learners who select this answer.}}
   Wrong Answer {{Feedback for learners who select this answer.}}
   (Correct Answer) {{Feedback for learners who select this answer.}}
+  ]]
+
+.. note:: When you include feedback, you might find it more convenient use new
+  lines to to separate the answer options.
```
Comment (muzaffaryousaf) [inline en_us/shared/exercises_tools/dropdown.rst]: `convenient use new lines to to separate` --> `convenient to use new lines to separate`

**Comment by muzaffaryousaf** [comment]
@lamagnifica I'm done reviewing this and left some comments. Thanks.

File: en_us/shared/exercises_tools/dropdown.rst (hunk 2 above)
Comment (lamagnifica) [inline en_us/shared/exercises_tools/dropdown.rst]: hmm, is there a sandbox you can point me to? I thought I tested these all out, but apparently I wasn't getting the right results...

**Comment by lamagnifica** [comment]
Reviewers I have tried to clean up the OLX. Please take a look. HTML version up to date.

File: en_us/shared/exercises_tools/dropdown.rst (hunk 2 above)
Comment (lamagnifica) [inline en_us/shared/exercises_tools/dropdown.rst]: @muzaffaryousaf I did test, and these problem work both ways: with the label before the optionsresponse, and with the label after the option response (!)

**Comment by lamagnifica** [comment]
All edits made to date incorporated.

File: en_us/shared/exercises_tools/dropdown.rst [hunk 9]
```diff
@@ -6,176 +6,180 @@ Dropdown Problem
...
 
-**************************************************
+In this example, a single problem component contains multiple questions, all of
+them using the dropdown problem type. To add the example illustrated
+above, you enter the following text and Markdown formatting in the simple
+editor in Studio.
+
+::
+
+  What type of data are the following?
```
Comment (sstack22) [inline en_us/shared/exercises_tools/dropdown.rst]: @lamagnifica - I'm not sure that this question is fully accessible, as the primary question does not have a label (chatted with @cptvitamin as well). I think we can either update this problem based on Mark's suggestions (I'm not sure how to write it correctly), or simplify it.

To simplify it we can just have 1 question, such as >>What type of data is age?<< and remove the other questions.

Comment (lamagnifica) [reply en_us/shared/exercises_tools/dropdown.rst]: I actually made each one a separate, complete question.

**Comment by muzaffaryousaf** [comment]
@lamagnifica Any plan to add new screen shots instead of deleted ones ?

**Comment by muzaffaryousaf** [comment]
👍 LGTM besides one question.

**Comment by lamagnifica** [comment]
@muzaffaryousaf we are trying to reduce the number of screen shots included in the docs since they introduce a lot of overhead to keep up to date (and cost to translators). So I replaced one screen shot, but then removed the others for each of the problem types.

**Pull request merged**